gokart/zip_client.py

```diff
@@ -62,13 +69,12 @@ def exists(self) -> bool:
 
     def make_archive(self) -> None:
         extension = os.path.splitext(self._file_path)[1]
-        shutil.make_archive(
-            base_name=self._temporary_directory, format=extension[1:], root_dir=self._temporary_directory)
+        shutil.make_archive(base_name=self._temporary_directory, format=extension[1:], root_dir=self._temporary_directory)
```

Comment:
How about using Zipfile package for compression? For upzip, you already use Zipfile.

I think it's better using same package for simplicity. Or are there any reason for using `shutil`?

Reply:
It sounds good. I fix codes to use zipfile to compress files.

Reply:
@hirosassa I tried to use zipfile instead of shutil It found that code for compression with zipfile is like complicated. Please see https://stackoverflow.com/questions/1855095/how-to-create-a-zip-archive-of-a-directory . So I think it is better to use shutil to compress directories until problems occur.

Reply:
Understood! Thanks for investigating!! LGTM